recipe/meta.yaml

```diff
@@ -22,12 +28,14 @@ requirements:
 
   host:
     - python
+    - libitk
     - libitk-devel
     - libsimpleitk ={{ version }}
     - fftw
 
   run:
     - vs2015_runtime  # [win]
+    - {{ pin_compatible('libitk', exact=True) }}
```

Comment:
If every things worked 100% the libitk runtime would be a transitive dependency.

The below libitk-devel run time dependency certainly should bring in libitk for the run time. The libitk-devel should not be required at run time, and it would certainly make the install smaller if it was not needed, but likely the additional dependency resolved some issue.

Reply:
Yes, but both are needed at runtime (at least on Windows, see #22), so I prefer to be explicit about what's needed to save debugging time in the future. We can investigate how to get this to a "100%" in a future PR!